### DISCUSSION TITLE
Getting an error while updating to Quartz .NET 3.1.0

### DISCUSSION THREAD

**Discussion by ektagoyal98** [quartznet/quartznet#933] [General]
In our project we need to update the Quartz library with version 3.1.0 and When I tried to update the library I am facing an issue.

![Error](https://user-images.githubusercontent.com/69619977/90102468-4905c280-dd5e-11ea-948a-3ad2bf621c28.PNG)
In our project we are currently using .NET framework v4.7.1. I observed that Quartz lib version 3.1 is backward compatible with this version but it need the reference of library Microsoft.Extensions.Logging.Abstractions -Version 3.1.6.

When I update the Logging.Abstractions it requires the version .NETStandard 2.0 which is not backward compatible and without updating the same I can not update Quartz library.

So do we have any workaround for same. If yes , Please help me by sharing the same.

**Comment by lahma** [comment]
The dependencies have been [downgraded already in master](https://github.com/quartznet/quartznet/pull/923/files). This change will be part of 3.2 release which will be released at some point, but you can already get them from [master MyGet feed](https://www.myget.org/F/quartznet/api/v3/index.json).

**Reply by ektagoyal98** [reply]
Microsoft.Extensions.Logging.Abstractions with version 2.1.1 which we have downgraded also support .NETStandard 2.0 only.

![Logger 2 1 1 dependency](https://user-images.githubusercontent.com/69619977/90124649-60ed3e80-dd7e-11ea-8b64-594f0350cae8.PNG)

**Comment by lahma** [comment]
.NETStandard 2.0 is compatible with full framework: https://docs.microsoft.com/en-us/dotnet/standard/net-standard

**Reply by ektagoyal98** [reply]
I am getting same error for 2.1.1 also.

Could not install package 'Microsoft.Extensions.Logging.Abstractions 2.1.1'. You are trying to install this package into a project that targets '.NETFramework,Version=v4.7.1', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.
